### ISSUE TITLE
Agents getting stuck on "getTask"

### ISSUE THREAD

**Issue by alienhorse** [hashtopolis/server#717]
I am looking for some help....

My agents are getting stuck on "getTask" with "Keyspace measure failed!' error.

I am running a simple brute force attack on MD5 hashes with the following command to test the system.

#HL# -a 3 ?u?l?l?l?l?l?d?d -w 4 - o

As for agents, I am using Vast.ai to rent GPU with the following container:

https://hub.docker.com/r/milz0/hashtopolis-hashcat-vast

I tried shortening the mask to six characters only but getting the same error.

**Comment by alienhorse** [comment]
After doing some troubleshooting, I got a little better understanding of the situation.

It looks like the agents are going into this error "ATTENTION! No OpenCL-compatible or CUDA-compatible platform found."

Typically, I can disable this error in hashcat by using this command  --backend-ignore-cuda

However, this command doesn't seem to work in hashtopolis unless I have to add this command somewhere specific. 

I have tried adding this command in both Attack Command in Tasks and Extra parameters in Agents like below:

Attack Command: -a 3 #HL# ?u?l?l?l?l?l?d?d -w 4 --backend-ignore-cuda
Extra parameters: --backend-ignore-cuda

Since I am mainly renting crackers from Vast.ai, updating the CUDA is not possible on those devices.

I would appreciate it if anyone can help guide me with this error.
